===================================================================
RCS file: /cvsroot/hibernate/Hibernate2/src/net/sf/hibernate/dialect/SQLServerDialect.java,v
retrieving revision 1.9
retrieving revision 1.10
diff -u -r1.9 -r1.10
--- hibernate/Hibernate2/src/net/sf/hibernate/dialect/SQLServerDialect.java	2004/08/06 09:50:42	1.9
+++ hibernate/Hibernate2/src/net/sf/hibernate/dialect/SQLServerDialect.java	2005/02/22 14:05:52	1.10
@@ -1,12 +1,19 @@
-//$Id: SQLServerDialect.java,v 1.9 2004/08/06 09:50:42 oneovthafew Exp $
+//$Id: SQLServerDialect.java,v 1.10 2005/02/22 14:05:52 oneovthafew Exp $
 package net.sf.hibernate.dialect;
 
+import java.sql.Types;
+
 /**
  * A dialect for Microsoft SQL Server 2000
  * @author Gavin King
  */
 public class SQLServerDialect extends SybaseDialect {
 
+	public SQLServerDialect() {
+		registerColumnType( Types.VARBINARY, "image" );
+		registerColumnType( Types.VARBINARY, 8000, "varbinary($l)" );
+	}
+	
 	public String getLimitString(String querySelect, boolean hasOffset, int limit) {
 		if (hasOffset) throw new UnsupportedOperationException("sql server has no offset");
 		return new StringBuffer( querySelect.length()+8 )
